src/components/features/action/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+
+import { IoLogoYoutube } from "react-icons/io";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be before the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine